README.md

```diff
@@ -8,4 +8,4 @@ This repository contains several examples which explain how to get started with
 
 ## Documentation
 
-[Docs](./docs/README.md)
+For documentationtion read [here](https://docs.saucelabs.com/visual-testing/sauce-visual/index.html).
```

Comment:
@KunalJain , @marstupa 
Great that we have a link! I checked the content and it looks more like overview how things works and idea behind the product. I was wondering that maybe we should communicate it: 
`Learn more how to use UI and our API here`
What do you think guys?

Reply:
How about this, to make it more actionable:
Visit our Documentation to learn how to use the UI and API for Visual Testing.